features/tree/handlers.go

```diff
@@ -74,28 +87,36 @@ func (f Feature) GetTree(c *gin.Context) {
 			select descendant as object_id from Paths 
 			where ancestor = ? and path_length != 0 and path_length <= ?
 		);`, rootObjectId, depth)
-
 	if err != nil {
 		c.AbortWithError(http.StatusBadRequest, err)
 		return
 	}
 
+	// ObjectRow represents a row in the object database table
 	type ObjectRow struct {
-		ObjectId       uint64
-		ParentObjectId uint64
-		ObjectKey      string
-		SizeBytes      uint64
-		ObjectType     string
-		TakenAtSec     sql.NullInt64
-		ObjectsInside  sql.NullInt64
-		LockedUntilSec sql.NullInt64
+		ObjectId       uint64        `json:"objectId"                 example:"123"`
+		ParentObjectId uint64        `json:"parentObjectId"           example:"456"`
+		ObjectKey      string        `json:"objectKey"                example:"object_key"`
+		SizeBytes      uint64        `json:"sizeBytes"                example:"102400"`
+		ObjectType     string        `json:"objectType"               example:"image"`
+		TakenAtSec     sql.NullInt64 `json:"takenAtSec,omitempty"     example:"1630342017"`
+		ObjectsInside  sql.NullInt64 `json:"objectsInside,omitempty"  example:"2"`
+		LockedUntilSec sql.NullInt64 `json:"lockedUntilSec,omitempty" example:"1630343017"`
```

Comment:
Thank you for adding `json` tags, because I didn't care about them. It is the right move and I should've done it earlier.

One thing to consider is if you are breaking an API contract by doing this change. If `ObjectRow` is returned somewhere in controller as response, by doing this you will break frontend, because it expects JSON keys, which starts from upper case.

It is not a big deal on our project, since we can easily change every part of the system. Just keep the client in mind when working with APIs.

Also in some sense, changing fields keys is not related to adding documentation. This makes the commit non atomic. I will close my eyes on this one, cause it is minor IMO. 

TODO:
- Change keys in SPA or make them start with uppercase to not break a contract